bgpd/bgp_packet.c

```diff
@@ -982,6 +982,7 @@ static void bgp_notify_send_internal(struct peer_connection *connection,
 		peer->notify.code = bgp_notify.code;
 		peer->notify.subcode = bgp_notify.subcode;
 		peer->notify.length = bgp_notify.length;
+		peer->notify.hard_reset = hard_reset;
```

Comment:
Do you need this line?

Reply:
It's a separate commit but needed for tests.